components/Hero/Hero.tsx

```diff
@@ -227,7 +228,7 @@ export default function Hero() {
                 zIndex: 1,
                 background: theme.palette.background.default,
               }}
-              sx={{ width: { xs: "90%", sm: "75%" } }}
+              sx={{ width: { xs: "90%", sm: "88%", md: "85%" } }}
```

Comment:
Try to use rounded values as much as possible unless a design specification requires otherwise. Although this may not be necessary in this case, it will help in calculating widths for other elements on the same line